code/game/objects/items/devices/organ_module/active/armguns.dm

```diff
@@ -21,3 +21,10 @@
 	allowed_organs = list(BP_R_ARM, BP_L_ARM)
 	holding_type = /obj/item/gun/projectile/shotgun/pump/hunter_crossbow_implanted
 	price_tag = 800
+
+/obj/item/organ_module/active/simple/lasercannon_lasercannon
+	name = "embedded \"Solaris\" laser cannon'"
+	desc = "A laser cannon designed to be inserted into an arm. Typically installed into combat type synthics."
+	verb_name = "Deploy laser cannon SMG"
+	icon_state = "si_lasercannon"
+	holding_type = /obj/item/gun/energy/lasercannon/rnd
```

Comment:
shouldn't this start with some kind of power cell? Unless I'm mistaken it wont right? The blackshield synthetic taser for example starts with a robust-x. And the blackshield synthetic SMG is self charging.